corehq/apps/users/static/users/js/mobile_workers.ng.js

```diff
@@ -379,6 +379,13 @@
                 ctrl.$validators.validatePassword = function (password) {
                     if (!password) {
                         return false;
+                    } else if (!(
+                        password.length >= 8 &&
+                        /\W/.test(password) &&
+                        /\d/.test(password) &&
+                        /[A-Z]/.test(password)
```

Comment:
is the requirement that it needs a lower case and an upper case? weird that it just wants an upper case letter